web/src/components/LeftPane/LeftPane.module.css

```diff
@@ -0,0 +1,46 @@
+.leftpane {
+  background-color: var(--color-light);
+  box-shadow: 2px 0 5px var(--color-grey);
+  flex-direction: column;
+  align-items: center;
+  justify-content: start;
+  display: flex;
+  padding: 1rem;
+  width: 400px;
+  height: 100vh;
```

Comment:
The height: 100vh could cause issues on mobile - it might be better to set the `min-height` here instead